tools/ci_build/github/azure-pipelines/templates/rocm.yml

```diff
@@ -0,0 +1,154 @@
+          --privileged \
+          --ipc=host \
+          --network=host \
+          --cap-add=SYS_PTRACE \
+          --security-opt seccomp=unconfined \
+          -e CC=/opt/rh/devtoolset-10/root/usr/bin/cc -e CXX=/opt/rh/devtoolset-10/root/usr/bin/c++ -e CFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all" -e CXXFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all" \
+          --volume $(Build.SourcesDirectory):/onnxruntime_src \
+          --volume $(Build.BinariesDirectory):/build \
+          --workdir /onnxruntime_src \
+          --entrypoint $(PythonManylinuxDir)/bin/python3 \
+          -e NIGHTLY_BUILD \
+          -e BUILD_BUILDNUMBER \
+          --user onnxruntimedev \
+          onnxruntimetrainingrocmbuild-rocm${{ parameters.RocmVersion }} \
+            /onnxruntime_src/tools/ci_build/build.py \
+              --config Release \
+              --use_rocm \
+              --rocm_version=${{ parameters.RocmVersion }} \
+              --rocm_home=/opt/rocm \
+              --nccl_home=/opt/rocm \
+              --update \
+              --parallel \
+              --build_dir /build \
+              --build \
+              --build_wheel \
+              --skip_tests \
+              --enable_training \
+              --cmake_extra_defines onnxruntime_BUILD_UNIT_TESTS=OFF \
+              --enable_training_torch_interop
+      workingDirectory: $(Build.SourcesDirectory)
+    displayName: 'Build onnxruntime (in container)'
+
+  # All UTs were here are now covered in AMD CI - see orttraining-pai-ci-pipeline.yml
+  # This CI is mainly responsible for packaging. The uploaded whl could be used in the downstream CIs (if any).
+  # For example, docker image build (e.g., PTCA), reporting CI, etc. to further verify the whl as needed.  
+  # To view the UTs disabled from this CI - see https://github.com/microsoft/onnxruntime/pull/11945 for examples
+
+  - script: |-
+      echo "Tests will run using HIP_VISIBLES_DEVICES=$HIP_VISIBLE_DEVICES"
+      video_gid=$(getent group | awk '/video/ {split($0,a,":"); print(a[3])}')
+      echo "Found video_gid=$video_gid; attempting to set as pipeline variable"
+      echo "##vso[task.setvariable variable=video]$video_gid"
+      render_gid=$(getent group | awk '/render/ {split($0,a,":"); print(a[3])}')
+      echo "Found render_gid=$render_gid; attempting to set as pipeline variable"
+      echo "##vso[task.setvariable variable=render]$render_gid"
+    displayName: 'Find video and render gid to be mapped into container'
+
+  - task: CopyFiles@2
+    displayName: 'Copy Python Wheel to: $(Build.ArtifactStagingDirectory)'
+    inputs:
+      SourceFolder: '$(Build.BinariesDirectory)'
+      Contents: 'Release/dist/*.whl'
+      TargetFolder: '$(Build.ArtifactStagingDirectory)'
+
+  - task: CmdLine@2
```

Comment:
The task looks unnecessary. Please check the mail Xavier Dupre replied on May 18th

Reply:
I would prefer to keep it unchanged. Even if the task is unnecessary, should the owner of this pipeline to delete it.